examples/minimal/emr-cluster.tf

```diff
@@ -58,21 +58,23 @@ module "sg-ports-emr" {
 }
 
 module "aws-emr-sg-master" {
-  source              = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
-  vpc_id              = var.vpc_id
-  ingress_cidr_blocks = var.ingress_cidr_blocks
-  egress_cidr_blocks  = var.egress_cidr_blocks
-  ingress_ports       = module.sg-ports-emr.ingress_master_ports
-  sg_name_prefix      = format("%s-%s", var.name_prefix, "emr-master")
-  egress_protocol     = "all"
-  ingress_protocol    = "tcp"
-  tags                = merge(var.tags, var.emr_tags)
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = module.aws-sg-vm.security_group_ids
+  egress_cidr_blocks      = var.egress_cidr_blocks
+  ingress_ports           = module.sg-ports-emr.ingress_master_ports
+  sg_name_prefix          = format("%s-%s", var.name_prefix, "emr-master")
+  egress_protocol         = "all"
+  ingress_protocol        = "tcp"
+  tags                    = merge(var.tags, var.emr_tags)
 }
 
 module "aws-emr-sg-core" {
   source              = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
   vpc_id              = var.vpc_id
   ingress_cidr_blocks = var.ingress_cidr_blocks
+  ingress_security_groups = module.aws-sg-vm.security_group_ids
```

Comment:
super-nit picky but you'll want to run `terraform fmt` here and a few other places